pyproject.toml

```
@@ -72,4 +72,5 @@ dev = [
     "pytest-env==1.1.3",
     "pytest-flask==1.3.0",
     "pytest-mock==3.14.0",
+    "ruff>=0.7.4",
```

We now pin to exact versions of things in applications/services (ie not libraries - read: utils). Renovate will bump appropriately as needed, but it feels clearer to say "we expect our application to have these versions of things"

```suggestion
    "ruff==0.7.4",
```